### PR TITLE
Update Squirrel Nuget Package for building on the build server

### PR DESCRIPTION
**Pull request by PieboyMcGillicuddy**
1. Modify projects to point to VS 2015 build tools
2. Fix a few compiler errors
3. Change .gitmodules to use SSH for build server.

### CODE REVIEW

File: src/Update/Update.csproj [hunk 1]
```diff
@@ -38,14 +38,17 @@
...
     </Reference>
-    <Reference Include="DeltaCompressionDotNet.MsDelta">
-      <HintPath>..\..\packages\DeltaCompressionDotNet.1.1.0\lib\net45\DeltaCompressionDotNet.MsDelta.dll</HintPath>
+    <Reference Include="DeltaCompressionDotNet.MsDelta, Version=1.1.0.0, Culture=neutral, PublicKeyToken=46b2138a390abf55, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\DeltaCompressionDotNet.1.1.0\lib\net20\DeltaCompressionDotNet.MsDelta.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="DeltaCompressionDotNet.PatchApi">
-      <HintPath>..\..\packages\DeltaCompressionDotNet.1.1.0\lib\net45\DeltaCompressionDotNet.PatchApi.dll</HintPath>
+    <Reference Include="DeltaCompressionDotNet.PatchApi, Version=1.1.0.0, Culture=neutral, PublicKeyToken=3e8888ee913ed789, processorArchitecture=MSIL">
```
Comment (teh173) [inline src/Update/Update.csproj]: so you didn't find a way for this to be net45?

File: test/packages.config [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DeltaCompressionDotNet" version="1.0.0" targetFramework="net45" />
+  <package id="DeltaCompressionDotNet" version="1.1.0" targetFramework="net45" />
```
Comment (teh173) [inline test/packages.config]: should this match the other net20..?

**Comment by teh173** [comment]
Also: Updated Squirrel build to use nuspec. http://teamcity.techsmith.com/admin/editBuildParams.html?id=buildType:DotNetCommonWindows_Squirrel_CreateNuGetPackage2015

**Review by dmarkachev — APPROVED** [review]
:+1:

**Pull request merged**